Specs/Scene/ExpressionSpec.js

```diff
@@ -529,4 +529,111 @@ defineSuite([
         expression = new Expression(new MockStyleEngine(), '(!(1 + 2 > 3)) ? (2 > 1 ? 1 + 1 : 0) : (2 > 1 ? -1 + -1 : 0)');
         expect(expression.evaluate(undefined)).toEqual(2);
     });
+
+    it('evaluates member expression', function() {
+        var feature = new MockFeature();
+        feature.addProperty('height', 10);
+        feature.addProperty('width', 5);
+        feature.addProperty('string', 'hello');
+        feature.addProperty('boolean', true);
+        feature.addProperty('color', Color.RED);
+        feature.addProperty('color.red', 'something else');
+        feature.addProperty('feature.color', Color.GREEN);
+        feature.addProperty('feature', {
+            color : Color.BLUE
+        });
+        feature.addProperty('null', null);
+        feature.addProperty('undefined', undefined);
+
+        var expression = new Expression(new MockStyleEngine(), '${color.red}');
+        expect(expression.evaluate(feature)).toEqual(1.0);
+
+        expression = new Expression(new MockStyleEngine(), '${color.blue}');
+        expect(expression.evaluate(feature)).toEqual(0.0);
+
+        expression = new Expression(new MockStyleEngine(), '${height.blue}');
+        expect(expression.evaluate(feature)).toEqual(undefined);
+
+        expression = new Expression(new MockStyleEngine(), '${undefined.blue}');
+        expect(expression.evaluate(feature)).toEqual(undefined);
+
+        expression = new Expression(new MockStyleEngine(), '${feature.color}');
+        expect(expression.evaluate(feature)).toEqual(Color.RED);
+
+        expression = new Expression(new MockStyleEngine(), '${feature.feature.color}');
+        expect(expression.evaluate(feature)).toEqual(Color.BLUE);
+
+        expression = new Expression(new MockStyleEngine(), '${feature.color.red}');
+        expect(expression.evaluate(feature)).toEqual(1.0);
+
+        expect(function() {
```

Comment:
Generally, we separate the exception tests like here and below into separate tests.